src/routes.js

```diff
@@ -105,6 +106,7 @@ export default (
             </RequireLogin>
           )}
         />
+        <Route path="/discover-more-people" render={() => <DiscoverMorePeople />} />
```

Comment:
I'm not a big fan of such URLs. I think that calling it `discover` or `discover-people` would be enough.

Reply:
yeah I'll change it to `discover`